config/routes.rb

```diff
@@ -4,6 +4,13 @@
   resources :categories
   resources :lessons
 
+  resources :words do
+    collection do
+      get "dashboard"
+      get "categories"
+    end
+  end
+
```

Comment:
instead of this you can use this instead:

```
resources :categories do
    member do
      resources :words
    end
  end
```

this code will make sure that a word will always belong to a certain category